prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/project/ProjectAccessRequestWorkFlowTask.java

```diff
@@ -18,6 +18,8 @@
 import lombok.NonNull;
 import lombok.extern.slf4j.Slf4j;
 
+import org.springframework.http.ResponseEntity;
+
 import static com.redhat.parodos.tasks.project.consts.ProjectAccessRequestConstant.ACCESS_REQUEST_APPROVAL_USERNAMES;
 import static com.redhat.parodos.tasks.project.consts.ProjectAccessRequestConstant.ACCESS_REQUEST_ESCALATION_USERNAME;
 import static com.redhat.parodos.tasks.project.consts.ProjectAccessRequestConstant.ACCESS_REQUEST_ID;
```

Comment:
nit: static import are not allowed in production code according to [code style doc](https://github.com/spring-projects/spring-framework/wiki/Code-Style)

Reply:
Fixed.